tests/DataTransferObjects/LogLineTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Auditing\DataTransferObjects;
+
+use DateTime;
+use LoyaltyCorp\Auditing\DataTransferObjects\LogLine;
+use Tests\LoyaltyCorp\Auditing\TestCase;
+
+/**
+ * Class LogLineDtoTest
+ *
+ * @covers \LoyaltyCorp\Auditing\DataTransferObjects\LogLine
+ */
+class LogLineTest extends TestCase
+{
+    /**
+     * Test getters
+     *
+     * @return void
+     *
+     * @throws \Exception
+     */
+    public function testGetters(): void
+    {
+        $instance = $this->getInstance();
+        self::assertSame('127.0.0.1', $instance->getClientIp());
+        self::assertEquals(new DateTime('2019-05-20T11:12:13Z'), $instance->getOccurredAt());
+        self::assertSame(['customer_id' => 11], $instance->getRequestData());
+        self::assertSame(['customer_name' => 'John'], $instance->getResponseData());
+        self::assertSame(1, $instance->getStatus());
+    }
+
+    /**
+     * Test toArray() method
+     *
+     * @return void
+     *
+     * @throws \Exception
+     */
+    public function testToArray(): void
+    {
+        // Cannot use assertSame() as there's object inside array.
+        self::assertEquals(
+            [
+                'clientIp' => '127.0.0.1',
+                'occurredAt' => new DateTime('2019-05-20T11:12:13Z'),
```

Comment:
Should be getting the formatted Zulu time here.

Reply:
changed